src/ReactiveDomain/ReactiveDomain.Tests/Logging/when_logging_disabled_commands_fired.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using Xunit;
+using ReactiveDomain.Bus;
+using ReactiveDomain.EventStore;
+using ReactiveDomain.Messaging;
+using ReactiveDomain.Tests.Helpers;
+using ReactiveDomain.Tests.Subscribers.QueuedSubscriber;
+using Xunit.Sdk;
+
+namespace ReactiveDomain.Tests.Logging
+{
+
+    // ReSharper disable once InconsistentNaming
+    public class when_logging_disabled_commands_fired :
```

Comment:
This is a confusing name. I got all the way through reading this class thinking "I don't see a 'disable logging' command getting fired here" and then I got to the next class and realized what the name meant. Perhaps a more clear name is in order.

Reply:
when_logging_is_disabled_and_commands_are_fired?

Reply:
That works

Reply:
This hasn't been updated. There are other classes with similar names that should also be changed for clarity.

Reply:
done. Thought I'd already done it.